web/test/factories.ts

```diff
@@ -392,6 +392,17 @@ export const generateActiveUser = (overrides: Partial<ActiveUser>): ActiveUser =
   };
 };
 
+export const generateAddress = (overrides: Partial<Address>): Address => {
+  return {
+    addressLine1: `some-address-1-${randomInt()}`,
+    addressLine2: `some-address-2-${randomInt()}`,
+    addressMunicipality: undefined,
+    addressState: undefined,
+    addressZipCode: `0${randomIntFromInterval("07001", "08999")}`,
```

Comment:
[dust] I don't think we need the leading "0"s here

```suggestion
    addressZipCode: `0${randomIntFromInterval("7001", "8999")}`,
```